numpy/core/src/multiarray/multiarraymodule.c

```diff
@@ -1583,7 +1583,7 @@ _array_fromobject(PyObject *NPY_UNUSED(ignored), PyObject *args, PyObject *kws)
 
     if (PyTuple_GET_SIZE(args) > 2) {
         PyErr_SetString(PyExc_ValueError,
```

Comment:
```suggestion
        PyErr_Format(PyExc_ValueError,
```
since now it's a format string.